cmd/terramate/cli/cli_runner_test.go

```diff
@@ -70,7 +70,10 @@ func (ts tscli) run(args ...string) runResult {
 	if ts.chdir != "" {
 		allargs = append(allargs, "--chdir", ts.chdir)
 	}
-
+	if len(args) > 0 {
+		// Avoid failing test when calling just terramate with no args
+		allargs = append(allargs, "--log-level", "fatal")
+	}
```

Comment:
added #148 as alternative.... if no stderr/stdout is set expectation could be not to test for it?